### DISCUSSION TITLE
Authentication provider for token usage?

### DISCUSSION THREAD

**Discussion by HaoYang670** [apache/kyuubi#5053] [Q&A]
Hi! Is there a way to implement custom authentication for token usage? I see there is a `PasswdAuthenticationProvider` for username / password, but not find one for token (i.e. the Bearer)

**Comment by HaoYang670** [comment]
Hi @pan3793, do you know whether there is such one?

**Reply by pan3793** [reply]
Does Hive support that? The authentication framework should be similar as Hive

**Reply by HaoYang670** [reply]
Not sure. But as far as I know, Databricks supports token
